apps/site/eslint.config.js

```diff
@@ -98,7 +48,7 @@ export default tseslint.config(
   {
     files: ['**/*.{mdx,tsx}'],
     rules: {
-      '@typescript-eslint/consistent-type-definitions': ['error', 'type'],
+      // '@typescript-eslint/consistent-type-definitions': ['error', 'type'],
       'react/no-unescaped-entities': 'off',
```

Comment:
I'm not sure how to enable this rule. When I try, I get the error "Key '@typescript-eslint/consistent-type-definitions': Could not find plugin '@typescript-eslint'," but isn't the plugin enabled in the base config?

Reply:
I answered on another comment!